Dockerfile

```diff
@@ -1,6 +1,8 @@
 # Pull the base image with given version.
-ARG BUILD_TERRAFORM_VERSION=0.11.7
-FROM microsoft/terraform-test:${BUILD_TERRAFORM_VERSION}
+ARG BUILD_TERRAFORM_VERSION=0.12.9
+FROM hashicorp/terraform:${BUILD_TERRAFORM_VERSION}
```

Comment:
I'm not sure if you want to use your own base image. Having a base image makes testing faster but this could also just pull go and terraform onto an alpine image and test with that